src/main/java/com/atherys/towns/api/permission/town/TownPermissions.java

```diff
@@ -63,4 +63,7 @@
 
     // permission to destroy the town
     public static final TownPermission RUIN_TOWN = new TownPermission("atherystowns.town.ruin", "Ruin Town");
+
+    // permission to pay off taxes
+    public static final TownPermission DEBT_PAY = new TownPermission("atherys.town.debt.pay", "Pay back debt");
```

Comment:
Should probably follow the standard, maybe:
atherystowns.town.pay_debt

Reply:
oh, i'm not sure why I split up atherys and town like that lol, good catch.